module_generator/cookiecutter-template/hooks/post_gen_project.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import subprocess
+
+
+if __name__ == "__main__":
+    subprocess.run(
```

Comment:
I ran into an issue when trying this out due to the `post_gen_project` command running in the root directory of the generated project, so the `../generate_main.py` couldn't be found when generating a project outside of the `viam-python-sdk` directory: https://cookiecutter.readthedocs.io/en/stable/advanced/hooks.html#hook-execution

Could the `generate_main.py` be included in the `hooks` directory to be called without shelling out to execute?

Reply:
How did you generate a project outside of the `viam-python-sdk` directory? Right now, the way I set up the project, it should be generated within the `viam-python-sdk/cookiecutter-template` directory. Eventually, the end goal is to initialize a git repository from the template. This is the [epic](https://viam.atlassian.net/browse/RSDK-8550) if you'd like to see it.
When I put `generate_main.py` in the `hooks` directory and try to import the function, the script can't find the `generate_main` file.

Reply:
Ah, I just tried it while in the `viam-python-sdk` directory. I hate relative paths for imports 😩